src/games/strategy/engine/framework/map/download/DownloadFileParser.java

```diff
@@ -27,24 +27,23 @@ private DownloadFileParser() {}
 
   public static List<DownloadFileDescription> parse(final InputStream is) {
     @SuppressWarnings("unchecked")
-    final List<Map<String, Object>> yamlData = (List<Map<String, Object>>) (new Yaml()).load(is);
+    final List<Map<String, String>> yamlData = new Yaml().loadAs(is, List.class);
```

Comment:
Please add a comment that we know that the YAML structure is a list of Strings. We're adding a new assumption that there are no complex data structures in the yaml beyond a list of scalars.